### PR TITLE
Add Oracle Eloqua connector

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
# Description

Add Oracle Eloqua connector

Fixes https://github.com/ballerina-platform/ballerina-extended-library/issues/248

One line release note: 
- Add Oracle Eloqua connector

## Type of change

- [X] New feature (non-breaking change which adds functionality)

**Test Configuration**:
* Ballerina Version: Ballerina Swan Lake Beta 6
* Operating System: Ubuntu
* Java SDK: Java 11

# Checklist:

### Changes to OpenAPI definition

- [X] Modified `info` section of the OpenAPI definition file - [Example](https://github.com/ballerina-platform/ballerina-extended-library/discussions/74)

    - `description` - what the connector is about 
    - `x-ballerina-init-description` OpenAPI extension - connector initialization details 
    - `x-ballerina-display` OpenAPI extension - connector name and path to the connector icon 

- [X] Added proper description each API Key, if API Key authentication is defined in OpenAPI definition file 

### Changes to connector module

- [X] Added license header at the top of each .bal file. 
- [x] Added Package.md as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/77)
- [x] Added Module.md for the module of the  connector as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/78)
- [X] Added connector icon to the root of the connector. Icon needs to be a `png` of 200x200 px size with name `icon.png`
- [X] Added Ballerina.toml file with following information. For keywords refer to [guide](https://github.com/ballerina-platform/ballerina-extended-library/discussions/72)

### Verifying connector module

- [X] Compiled the connector successfully with the targeted ballerina version. 
- [X] Conducted smoke tests on the connector (Recommended when OpenAPI definition is obtain from an unofficial source)

### Security checks
 - [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: openapi/eloqua/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[package]
+distribution = "slbeta6"
+license = ["Apache-2.0"]
+keywords = ["Marketing/Marketing Automation", "Cost/Paid"]
+org = "ballerinax"
+name = "eloqua"
+icon = "icon.png"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.1"
```
Comment (aneeshafedo) [inline openapi/eloqua/Ballerina.toml:9]: What is the version we need to give for newly added connectors ?

Comment (LakshanSS) [reply openapi/eloqua/Ballerina.toml:9]: Yes. Usually we add `1.0.0` for the new connectors.

Comment (SkNuwanTissera) [reply openapi/eloqua/Ballerina.toml]: Updated to `1.0.0`

**Comment by abeykoon** [comment]
We'll add a letter for the icon and release the connector. Oracle is too much restrictive on trademarks.

**Comment by SkNuwanTissera** [comment]
Updated PR with an new icon (word-mark)

**Review by RolandHewage — APPROVED** [review]

**Review by RolandHewage — APPROVED** [review]

File: openapi/eloqua/Package.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Connects to [Oracle Eloqua API](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-develop/Developers/GettingStarted/Tutorials/tutorials.htm) from Ballerina
+
+## Package overview
+The `ballerinax/eloqua` is a [Ballerina](https://ballerina.io/) connector for Oracle Eloqua API. This package provides the capability to extend the functionality of the product, build Apps, and perform high volume data transfers.
+
+### Compatibility
+|                    | Version          |
+|--------------------|------------------|
+| Ballerina Language |  Swan Lake Beta6 |
```
Comment (aneeshafedo) [inline openapi/eloqua/Package.md:9]: Missing API version

Comment (SkNuwanTissera) [reply openapi/eloqua/Package.md]: Updated with API version

**Comment by SkNuwanTissera** [comment]
Added `Quickstart` with a sample

**Review by aneeshafedo — APPROVED** [review]

**Pull request merged**